scripts/upgrade_version.py

```diff
@@ -0,0 +1,85 @@
+import click
+import os
+import subprocess
+import sys
+
+RAY_ABSOLUTE_PATH = ""
+
+NON_JAVA_FILES = set([
+    'ci/ray_ci/utils.py',
+    'python/ray/_version.py',
+    'src/ray/common/constants.h',
+])
+
+JAVA_FILES = set([
+    'java/api/pom_template.xml',
+    'java/performance_test/pom_template.xml',
+    'java/pom.xml',
+    'java/runtime/pom_template.xml',
+    'java/serve/pom_template.xml',
+    'java/test/pom_template.xml',
+])
+
+NON_JAVA_VERSION = "3.0.0.dev0"
+JAVA_VERSION = "2.0.0-SNAPSHOT"
+NEW_VERSION = "3.0.0.dev0"
+
+def update_current_version():
+    """
+    Scan for current Ray version and update the version global variables.
+    """
+    global NON_JAVA_VERSION
+    global JAVA_VERSION
```

Comment:
why use global? why not send in an arg?

Reply:
Sending args also works too! I can switch to that since it's easier to write unit tests on it.

Since these 2 variables are used for the `upgrade_version()` function as well, I thought it might be more convenient to use global variables so they can all reference without adding more arguments.

Reply:
fixed!